src/zcash/prf.cpp

```diff
@@ -2,7 +2,7 @@
 #include "crypto/sha256.h"
 
 uint256 PRF(bool a, bool b, bool c, bool d,
-            const uint256& x,
+            const uint252& x,
             const uint256& y)
 {
```

Comment:
The line `blob[0] &= 0x0F` below can be removed, I suggest replacing it with an assertion that the leading bits are zero.